src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const shortTerm = 3;
+  const shortTermDiscount = 20;
+
+  if (days >= longTerm) {
+    return days * pricePerDay - longTermDiscount;
+  }
+
+  if (days >= shortTerm) {
+    return days * pricePerDay - shortTermDiscount;
```

Comment:
There is a logical error here. The condition `days >= shortTerm` will always be true if `days >= longTerm`, which means the short-term discount will never be applied. You should use an `else if` statement to ensure that only one discount is applied.